bundles/remote_services/remote_services_api/include/remote_custom_serialization_service.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef CELIX_REMOTE_CUSTOM_SERIALIZATION_SERVICE_H
+#define CELIX_REMOTE_CUSTOM_SERIALIZATION_SERVICE_H
+
+#define RSA_CUSTOM_SERIALIZATION_SERVICE_NAME "rsa.custom.serialization"
+#define RSA_SERVICE_TARGETED_ID "targeted.service.id"
```

Comment:
I also think a service.id is wrong, because now the serialization always needs to figure out the service id of the target. Using a service property (list of types/messages supported by this bundle) would make more sense.